src/frontends/common/include/openvino/frontend/frontend.hpp

```diff
@@ -168,6 +168,16 @@ class FRONTEND_API FrontEnd {
     virtual InputModel::Ptr load_impl(const std::vector<ov::Any>& variants) const;
 
     void validate_path(const std::string& path) const;
+    void validate_path(const std::filesystem::path& path) const;
```

Comment:
Replace them by path version is not public member

Reply:
ok